### ISSUE TITLE
ProjectPasskeys: Add instructions for jans-fido2 localhost. 

### ISSUE THREAD

**Issue by yackermann** [JanssenProject/jans#9203]
Need to know how to run jans-fido2 locally with localhost. 

@yurem

**Comment by nynymike** [comment]
@yackermann  what certificates are you using for localhost? I thought FIDO requires trusted TLS...

**Comment by yackermann** [comment]
@nynymike TLS only required for normal deployment. FIDO works fine on localhost for testing and does not require any TLS.

None of the team was able to run tests locally, because the code wont even start with localhost, so some help from @yurem would be great

**Comment by moabu** [comment]
Hey @yackermann ,

@yurem wrote https://github.com/JanssenProject/jans/wiki/Local-run-under-Eclipse
and we have https://docs.jans.io/v1.1.4/contribute/development/#remote-debugging for remote debugging. 

@ossdhaval can we polish it and move Movchans doc in the docs. I think we also need to establish development docs regarding how to accomplish a quick dev instance a developer can work with

**Comment by ossdhaval** [comment]
Yes, I'll keep it in my TODO. To help me prioritize, do mention if there is a timeline to finish this.

**Comment by moabu** [comment]
Thanks!
~ 1 week
